### PR TITLE
Error messages for each field should be shown on separate lines #5085

### LINKED ISSUE #2352: The Exception messages need to be delimited  by line breaks (<br>) and not Const.EOL
Invalidity Info in statuses are now delimited by Const.EOL which is not rendered on a browser resulting in a huge paragraph of text. Either we can change EOL to a <br> or change statusMessage.jsp to replace all Const.EOL to '<br>'

### LINKED ISSUE #5085: Error messages for each field should be shown on separate lines
image

### PR DESCRIPTION
**Pull request by YongJieYongJie**
Fixes #5085 and fixes #2352

### CODE REVIEW

**Comment by YongJieYongJie** [comment]
Areas affected and fixed should include the following:
- Admin user when adding invalid instructor
- Admin user when saving invalid email
- Instructor when creating invalid feedback session
- Instructor when adding invalid instructors

// Areas where error messages are already displayed in separate lines (not exhaustive):
- Instructor when enrolling invalid  students

**Comment by wkurniawan07** [comment]
Fixes #2352 as well (duplicate issue).

**Comment by YongJieYongJie** [comment]
@JosephineKwa Ready for review!

File: src/main/java/teammates/ui/controller/Action.java [hunk 1]
```diff
@@ -513,9 +513,11 @@ protected ActionResult createImageResult(String blobKey) {
      * {@code isError} is also set to true.
      */
     protected void setStatusForException(Exception e) {
-        statusToUser.add(new StatusMessage(e.getMessage(), StatusMessageColor.DANGER));
         isError = true;
-        statusToAdmin = Const.ACTION_RESULT_FAILURE + " : " + e.getMessage();
+
+        String exceptionMessageForHtml = e.getMessage().replace(Const.EOL, "<br>");
```
Comment (JosephineKwa) [inline src/main/java/teammates/ui/controller/Action.java]: Perhaps you could extract the `<br>` into `Const`

**Comment by YongJieYongJie** [comment]
@JosephineKwa I extracted `"<br>"` as `Const.HTML_BR_TAG`

**Comment by JosephineKwa** [comment]
Good to merge

**Comment by thyageshm** [comment]
@JosephineKwa please merge this :-)

**Comment by ashrayjain** [comment]
@JosephineKwa @YongJieYongJie This merge is causing master to fail, both locally and on Travis. Specifically, InstructorCourseEnrollPageUiTest is failing at line 220.

**Comment by damithc** [comment]
@YongJieYongJie we have reverted this merge because it failed some testes. Submit another PR with the failing tests fixed.

**Pull request merged**